fast64_internal/sm64/c_templates/tile_scroll.py

```diff
@@ -0,0 +1,64 @@
+tile_scroll_c = '''#include <ultra64.h>
```

Comment:
why does this file have a .py extension if it's a C template?

Reply:
I didn't want to separate the `.c` and `.h` files to keep the repo's file count down, and make it obvious that these two files are linked. do you have a preferred method here?

Reply:
well at least not make it a .py (it definitely isn't python), maybe just .txt if .c doesn't work for you

Reply:
my bad it is python, I just can't read nvm